crates/rerun/src/run.rs

```diff
@@ -386,6 +401,31 @@ where
     }
 }
 
+fn initialize_thead_pool(threads_args: i32) {
+    // Name the rayon threads for the benefit of debuggers and profilers:
+    let mut builder = rayon::ThreadPoolBuilder::new().thread_name(|i| format!("rayon-{i}"));
+
+    if threads_args < 0 {
+        match std::thread::available_parallelism() {
+            Ok(cores) => {
+                let threads = cores.get().saturating_sub((-threads_args) as _).max(1);
+                re_log::debug!("Detected {cores} cores. Using {threads} compute threads.");
+                builder = builder.num_threads(threads);
+            }
+            Err(err) => {
+                re_log::warn!("Failed to query system of the number of cores: {err}");
```

Comment:
worth commenting that not setting `num_threads` is equivalent to setting it to zero, see https://docs.rs/rayon/latest/rayon/struct.ThreadPoolBuilder.html#method.num_threads